src/app/(protected)/account/settings/page.tsx

```diff
@@ -1,160 +1,5 @@
-                      placeholder="Email"
-                      isAboveError
-                      disabled
-                    />
-                  </FormGroupField>
-                </FormGroup>
-                <FormGroup>
-                  <FormGroupLabel>Password</FormGroupLabel>
-                  <FormGroupField>
-                    <Controller
-                      name="password"
-                      control={control}
-                      render={({ field, fieldState }) => (
-                        <PasswordField
-                          input={field}
-                          meta={{
-                            touched: !!fieldState.error,
-                            error: fieldState.error?.message,
-                          }}
-                          placeholder="Password"
-                          keyIcon
-                          isAboveError
-                        />
-                      )}
-                      rules={{ required: 'This is required field' }}
-                      defaultValue="initial password"
-                    />
-                  </FormGroupField>
-                </FormGroup>
-                <FormGroup>
-                  <FormGroupLabel>Mobile</FormGroupLabel>
-                  <FormGroupField>
-                    <FormField
-                      name="mobile"
-                      control={control}
-                      component="input"
-                      errors={errors}
-                      rules={{}}
-                      defaultValue=""
-                      placeholder="Mobile"
-                      isAboveError
-                    />
-                  </FormGroupField>
-                </FormGroup>
-                <FormGroup>
-                  <FormGroupLabel>Reference</FormGroupLabel>
-                  <FormGroupField>
-                    <FormField
-                      name="ref"
-                      control={control}
-                      component="input"
-                      errors={errors}
-                      rules={{
-                        required: 'This is required field',
-                      }}
-                      defaultValue="admin"
-                      placeholder="Reference"
-                      isAboveError
-                    />
-                  </FormGroupField>
-                </FormGroup>
-                <FormButtonToolbar>
-                  {/* @ts-ignore - Ignoring because of complex union types incorrectly inferred */}
-                  <Button variant="primary" type="submit">
-                    Submit
-                  </Button>
-                  <Button variant="secondary" type="button" onClick={() => {}}>
-                    Cancel
-                  </Button>
-                </FormButtonToolbar>
-              </FormContainer>
-            </CardBody>
-          </Card>
-        </Col>
-      </Row>
-    </Container>
-  );
-};
+const Setting = () => <SettingForm />;
```

Comment:
Can `export default SettingForm` work?

Reply:
Yes! Reafactored!

Reply:
refactor as export default function SettingPage(){}

Reply:
not exactly what I meant, but I'm okay (happier even) with this 👌